libs/firebase-utils/src/lib/firestoreMigrations/0072.ts

```diff
@@ -0,0 +1,39 @@
+import { Firestore } from '../types';
+import { runChunks } from '../firebase-utils';
+
+/**
+ * Update movies content type
+ * @param db
+ * @returns
+ */
+export async function upgrade(db: Firestore) {
+  const contentTypesToMovie = [
+    'animation', 
+    'feature_film',
+    'short',
+    'tv_film'
+  ]
+  const contentTypesToTV = [
+    'flow',
+    'series',
+    'performing_arts'
+  ]
```

Comment:
It's better to use mapping when possible to avoid `if`/`else` statements.
```suggestion
 const transform = {
   'animation': () => 'movie',
   'feature_film': () => 'movie',
   'short': () => 'movie',
   'tv_film': () => 'movie',
   'flow': () => 'tv',
   'series': () => 'tv',
   'performing_arts': () => 'tv',
   'documentary': (movie: Movie) => movie.runningTime.time < 60 ? 'tv' : 'movie'
 };
```